src/auth/dto/create-user.dto.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  IsString,
+  MinLength,
+  Matches,
+  IsNotEmpty,
+  IsEmail,
+  Length,
+} from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+export class CreateUserDto {
+  @IsString()
+  @ApiProperty({
+    example: 'username',
+  })
+  @IsNotEmpty({ message: 'First name is required' })
+  name: string;
```

Comment:
PullProofAI

The field `name` is labeled as "First name is required," but it seems to represent a username based on the example provided. Consider updating the label to reflect its purpose accurately.
```diff
-   @IsNotEmpty({ message: 'First name is required' })
+   @IsNotEmpty({ message: 'Username is required' })
```


<!-- This is an auto-generated comment by PullProofAI -->